app/src/main/java/com/kimdoori/githubapp/viewcommon/BaseViewHolder.kt

```diff
@@ -0,0 +1,24 @@
+package com.kimdoori.githubapp.viewcommon
+
+import android.view.LayoutInflater
+import android.view.ViewGroup
+import androidx.annotation.LayoutRes
+import androidx.databinding.DataBindingUtil
+import androidx.databinding.ViewDataBinding
+import androidx.recyclerview.widget.RecyclerView
+import com.kimdoori.githubapp.BR
+
+open class BaseViewHolder<ITEM_TYPE : Any>(
+    @LayoutRes layoutResourceId: Int,
+    parent: ViewGroup,
+    private val bindingVariableId: Int = BR.item
+) : RecyclerView.ViewHolder(
+    LayoutInflater.from(parent.context).inflate(layoutResourceId, parent, false)
```

Comment:
```suggestion
    LayoutInflater
        .from(parent.context)
        .inflate(layoutResourceId, parent, false)
```

여기도 함수 체이닝 컨벤션을 적용하면 좋을 것 같네요

Reply:
반영했습니다!